openapi/components/examples/content/200-pointer-changes.json

```diff
@@ -4,85 +4,19 @@
          {
             "entityType":"scene",
             "entityId":"QmWBMqDmFebLr4pJWxBpXdtToqTNxqeefzkerQ1C35cGeb",
-            "localTimestamp":1628614712569,
-            "changes":[
-               {
-                  "pointer":"-62,-100",
-                  "before":"QmSdyt9Sdys7Ntk7kM5Z8a6ASckyC7KuD1pWVNCU6bMA8e",
-                  "after":"QmWBMqDmFebLr4pJWxBpXdtToqTNxqeefzkerQ1C35cGeb"
-               }
-            ]
+            "localTimestamp":1628614712569
```

Comment:
Can we take the chance to add `pointers` to be compliant with the format used by snapshots?

Reply:
Okay, I like that. Should we add the authChain too? 🤔

Reply:
Oh yes, I think it is part of the answer already

Reply:
done